docs/source/usage/workflows/tracerParticles.rst

```diff
@@ -8,21 +8,78 @@ Tracer Particles
 Tracer particles are like :ref:`probe particles <usage-workflows-probeParticles>`, but interact self-consistenly with the simulation.
 They are usually used to visualize *representative* particle trajectories of a larger distribution.
 
+In PIConGPU each species can be a tracer particle species, this allows tracking fields along the particle trajectory.
```

Comment:
*... which allows ...* or make it a new sentence

Reply:
Should we call them `probes`, not `tracer` everywhere?

Reply:
Well, I like the conceptual separation between stationary probes that just get placed somewhere in an arbitrary distribution to record what's going on and the representatives of a moving, physical species that you can trace.
As long as it's clear that they both originate from the same concept of recording and just differ in what pusher they have.